scalableminds/webknossos
# avoid unnecessary rerendering if nothing changed

During the recent refactoring a bug was introduced that caused a `binary` to create an additional (unecessary) `plane2d` for the 3D-View, that always signalled to our render loop that it had changed. This caused a rerender in every frame even when doing nothing (13% CPU for the webknossos tab + another 15% for the GPU process). This PR fixes this and CPU usage while doing nothing is now down to 1-2% as expected.

Note: Flight mode also always renders with 60 FPS (since ... forever ^^) and has no code in place to avoid rerendering when nothing has changed. I'll create an issue to change that. - _Actually nvm, I did the change for flight mode as well!_

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Decrease CPU usage in all modes

Steps to test:
- Open tracing in orthogonal mode and do nothing. Open the Chrome Task Manager and check CPU usage.

Issues:
- fixes the HiWis electricity bills

------
